huxley/core/models.py

```diff
@@ -15,16 +15,23 @@
 
 
 class Conference(models.Model):
-    session         = models.PositiveSmallIntegerField(default=0)
-    start_date      = models.DateField()
-    end_date        = models.DateField()
-    reg_open        = models.DateField()
-    early_reg_close = models.DateField()
-    reg_close       = models.DateField()
-    min_attendance  = models.PositiveSmallIntegerField(default=0)
-    max_attendance  = models.PositiveSmallIntegerField(default=0)
-    open_reg        = models.BooleanField(default=True)
-    waitlist_reg    = models.BooleanField(default=False)
+    session          = models.PositiveSmallIntegerField(default=0, primary_key=True)
+    start_date       = models.DateField()
+    end_date         = models.DateField()
+    reg_open         = models.DateField()
+    early_reg_close  = models.DateField()
+    reg_close        = models.DateField()
+    min_attendance   = models.PositiveSmallIntegerField(default=0)
+    max_attendance   = models.PositiveSmallIntegerField(default=0)
+    open_reg         = models.BooleanField(default=True)
+    waitlist_reg     = models.BooleanField(default=False)
+    external         = models.CharField(max_length=128)
+    registration_fee = models.DecimalField(max_digits=6, decimal_places=2, default=Decimal('50.00'))
+    delegate_fee     = models.DecimalField(max_digits=6, decimal_places=2, default=Decimal('50.00'))
+
+    @classmethod
+    def get_conference(cls):
```

Comment:
nit: I think `get_current` is a more appropriate name (it's not clear what conference you're getting with `get_conference`).